gohelper.go

```
@@ -102,3 +360,37 @@ func getPlatformshRelationships() (Relationships, error) {
 
 	return rels, nil
 }
+
+// Map the variables environment variable string into the appropriate data structure.
+func extractVariables(vars string) (envList, error) {
+	jsonVars, _ := base64.StdEncoding.DecodeString(vars)
```

Ignoring errors is not a good idiom :)